tests/quick/test_generic_methods.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/python3
+"""
+Sanity test file for ncf methods
```

Comment:
```suggestion
Sanity test file for methods
```

Reply:
no big deal in tests but we should strive to remove references to "ncf" which does not make much sense